### PR TITLE
Use compressed blessed snapshots by default

### PR DESCRIPTION
**Pull request by jadeallenx**
Remove uncompressed snapshot parsing; always assume snapshots are compressed, load compressed data directly from JSON

### CODE REVIEW

**Review by jeffgrunewald — APPROVED** [review]

**Review by madninja — CHANGES_REQUESTED** [review]
Too much flexibility? Just set it to true in the blessed case.. No need for yet another variable

File: src/blockchain_worker.erl [hunk 1]
```diff
@@ -1116,7 +1116,9 @@ fetch_and_parse_latest_snapshot(SnapInfo0) ->
                            {undefined, _} -> #snapshot_info{};
                            {_, undefined} -> #snapshot_info{};
                            {BlessedHash, BlessedHeight} ->
-                               #snapshot_info{hash=BlessedHash, height=BlessedHeight}
+                               UseCompressed = application:get_env(blockchain, use_compressed_blessed_snapshots, true),
```
[the remark this answers is not in the document]
Comment (Vagabond) [reply src/blockchain_worker.erl:1119]: this does sound like a reasonable idea

**Comment by Vagabond** [comment]
IIRC you can open an uncompressed file as a compressed one and it will work fine

**Comment by jadeallenx** [comment]
> IIRC you can open an uncompressed file as a compressed one and it will work fine

yes this is true

**Review by jeffgrunewald — APPROVED** [review]

**Review by madninja — APPROVED** [review]

**Review by jeffgrunewald — APPROVED** [review]
🥳

**Pull request merged**